addons/l10n_fr_invoice_addr/views/report_invoice.xml

```diff
@@ -28,7 +28,7 @@
                 <t t-set="has_service" t-value="'service' in tax_scopes"/>
                 <t t-set="has_consu" t-value="'consu' in tax_scopes"/>
 
-                <div t-if="has_service or has_consu" class="col mb-2" name="operation_type">
+                <div t-if="has_service or has_consu" class="col col-3 mw-100 mb-2" name="operation_type">
```

Comment:
looking at the original view (`account.report_invoice_document`):

https://github.com/odoo/odoo/blob/b37dadc5d05645cad4483fba7849b4b35f0d20f2/addons/account/views/report_invoice.xml#L95

it would make sense to me to also have:

```py
t-attf-class="#{'col-auto col-3 mw-100' if report_type != 'html' else 'col'} mb-2"
```
If the report_type is not HTML, this would end up nearly to the same thing than your current proposition `col-auto col-3 mw-100 mb-2` with just `col-auto` instead of `col` as difference.

Reply:
Sounds good, made the correction